homeworkTest.js

```diff
@@ -0,0 +1,31 @@
+const array = ["4","5"];
+
+function testMap(arr){
+ let newArr = arr.map(item => item + "test");
+ return newArr;
+}
+
+function testReduce(arr){
+    return arr.reduce((a,b) => a.concat(b + 'test'),[])
+  }
+  
+console.log(testMap(array));
+console.log(testReduce(array));
+
+function greaterThenThreeFilter(arr){
+  return arr.filter(item => item > 3)
+}
+
+function greaterThenThreeReduce(arr){
+  let newArr = [];
```

Comment:
don't create a new array here, do it in the second argument of reduce, like you did in the first task